.github/workflows/main.yaml

```diff
@@ -7,6 +7,22 @@ on:
     - "*"
   pull_request:
 jobs:
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    container: golang:1.13.9-alpine3.11
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Test
+      run: |
+        apk add git
+        go get github.com/rakyll/statik
+        go generate
+        export CGO_ENABLED=0
+        go test ./...
```

Comment:
Can we use the same `scripts/alpine-setup.sh` script (I know it sets up upx too which is not needed), as in the Build step, to install bits in the image and then add this as `make test` target to Makefile, so that the tests can easily be run locally as well? (this target can depend on generate, and Makefile already has `CGO_ENABLED=0`)

Reply:
Sure :)